### ISSUE TITLE
Only 5.00 GB available. Mining suspended on device ...

### ISSUE THREAD

**Issue by 0x3025** [firoorg/firo#1225]
### Machine specs:
- OS:Windows10
- GPU:Nvidia RTX 2060

### It can't work
![image](https://user-images.githubusercontent.com/116722706/215442954-2856fca7-0382-44b6-9427-1f56acb8fa33.png)

**Comment by justanwar** [comment]
The current DAG size is 5.2GB. How much VRAM does this card has?

Is this GPU being used together with a display? It might explain why there's not enough memory available.

**Comment by LinuxPJC** [comment]
> The current DAG size is 5.2GB. How much VRAM does this card has?
> 
> Is this GPU being used together with a display? It might explain why there's not enough memory available.

### This card has 6GB VRAM.

###yep,It is being used together with display.

**Comment by justanwar** [comment]
Could you try running it in headless mode?

**Comment by LinuxPJC** [comment]
> Could you try running it in headless mode?

I don’t know how to running it in headless mode.

**Comment by justanwar** [comment]
See [this link](https://www.reddit.com/r/Ravencoin/comments/t3gfrd/1050ti_unable_to_mine_on_kawpow/) for possible solutions.

tl;dr: Windows is reserving some VRAM from the GPU, so you didn't get to use the full 6GB.
